header.go

```diff
@@ -652,6 +653,7 @@ func (h *ResponseHeader) DisableNormalizing() {
 // Reset clears response header.
 func (h *ResponseHeader) Reset() {
 	h.disableNormalizing = false
+	h.noDefaultContentType = false
```

Comment:
I'm wondering now if this should be done here or in `resetSkipNormalize()`. I don't think it matters much but maybe doing it in `resetSkipNormalize()` is cleaner as everything is reset there.